docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -818,6 +818,54 @@ include-tagged::{sql-specs}/docs/docs.csv-spec[formatDateTime]
 include-tagged::{sql-specs}/docs/docs.csv-spec[formatTime]
 --------------------------------------------------
 
+
+[[sql-functions-datetime-to_char]]
+==== `TO_CHAR`
+
+.Synopsis:
+[source, sql]
+--------------------------------------------------
+TO_CHAR(
+    date_exp/datetime_exp/time_exp, <1>
+    string_exp) <2>
+--------------------------------------------------
+
+*Input*:
+
+<1> date/datetime/time expression
+<2> format pattern
+
+*Output*: string
+
+*Description*: Returns the date/datetime/time as a string using the format specified in the 2nd argument. The formatting
+pattern conforms to
+https://www.postgresql.org/docs/13/functions-formatting.html[PostgreSQL Template Patterns for Date/Time Formatting].
+If any of the two arguments is `null` or the pattern is an empty string `null` is returned.
+
+[NOTE]
+If the 1st argument is of type `time`, then the pattern specified by the 2nd argument cannot contain date related units
+(e.g. 'dd', 'MM', 'YYYY', etc.). If it contains such units an error is returned.
+
+*Special Cases*
+
+- The result of the pattern `TZ`, `tz` and `OF` patterns is equivalent with, but differs from the results returned by the PostgreSQL implementation.
```

Comment:
`TZ` and `tz` (mostly) the short name of the timezone. PostgreSQL and Java has slightly different names for the different timezones. Sadly, I cannot just simply add a static mapping between the two, since the name of timezones  can also change over time. Did not seem to worth the effort. I can add more details explain this in the docs.

Also: I will remove the `OF` from this callout, because I was able to match it between Postgres and ES.